src/WpfMath/Atoms/RowAtom.cs

```diff
@@ -37,9 +37,7 @@ static RowAtom()
         public RowAtom(SourceSpan source, IList<TexFormula> formulaList)
             : this(
                 source,
-                formulaList
-                    .Where(formula => formula.RootAtom != null)
-                    .Select(formula => formula.RootAtom))
+                formulaList.Select(formula => formula.RootAtom))
```

Comment:
I have a better solution for this code: I'll delete it for good. The only place where it's called is an unused constructor of `TexFormula`.